src/makeRobotAccountant.js

```diff
@@ -6,7 +6,20 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
-}
+  let count = 1;
+
+  return function(a) {
+    return function(b) {
+      if (count % 2 === 0 && count !== 0 && count >= 4) {
```

Comment:
Count starting value = 1. You don't need to check if it's not 0. It can't be because you increase it by 1 with every iteration. Besides if you move count increase before if you can even start from 0, which is more logical.